discovery-provider/src/queries/queries.py

```diff
@@ -131,6 +131,32 @@ def get_tracks():
     return api_helpers.success_response(tracks)
 
 
+@bp.route("/track", methods=("GET",))
+def get_track():
+    track_title = request.args.get("title", type=str)
+
+    # TODO: need to decode title URL
+    if not track_title:
+        raise exceptions.ArgumentError("Invalid value for parameter 'title'")
+
+    track_id = request.args.get("id", type=int)
+    if not track_id:
+        raise exceptions.ArgumentError("Invalid value for parameter 'track_id'")
+
+    db = get_db()
+    with db.scoped_session() as session:
+        base_query = session.query(Track)
+        base_query = base_query.filter(Track.is_current == True, Track.title == track_title, Track.track_id == track_id)
+        query_results = paginate_query(base_query).all()
```

Comment:
@piazz I don't think we need this paginate_query(base_query).all() here. Instead I think we just want a single track, so it should just be ```base_query.one()```?